shelley/chain-and-ledger/executable-spec/src/Shelley/Spec/Ledger/Rewards.hs

```diff
@@ -539,9 +539,16 @@ rewardOnePool
               poolRA
               (Set.singleton (Reward LeaderReward (_poolId pool) lReward))
               mRewards
-          else mRewards
+          else
+            Map.insertWith
+              Set.union
+              poolRA
+              (Set.singleton (Reward LeaderReward (_poolId pool) (Coin 0)))
+              mRewards
```

Comment:
Can you explain why we do this?

Reply:
oh yes, thank you, that definitely deserves a comment. I will add one now!